src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/StaticWhitelistTest.java

```diff
@@ -125,7 +124,25 @@ static void sanity(URL definition) throws Exception {
             // Overrides CharSequence.isEmpty in Java 15+.
             new MethodSignature(String.class, "isEmpty", new Class<?>[0]),
             // Does not exist until Java 15.
-            new MethodSignature(CharSequence.class, "isEmpty", new Class<?>[0])
+            new MethodSignature(CharSequence.class, "isEmpty", new Class<?>[0]),
+            // Override the corresponding RandomGenerator methods in Java 17+.
+            new MethodSignature(Random.class, "nextBoolean", new Class<?>[0]),
+            new MethodSignature(Random.class, "nextBytes", new Class<?>[] {byte[].class}),
+            new MethodSignature(Random.class, "nextDouble", new Class<?>[0]),
+            new MethodSignature(Random.class, "nextFloat", new Class<?>[0]),
+            new MethodSignature(Random.class, "nextGaussian", new Class<?>[0]),
+            new MethodSignature(Random.class, "nextInt", new Class<?>[0]),
+            new MethodSignature(Random.class, "nextInt", new Class<?>[] {int.class}),
+            new MethodSignature(Random.class, "nextLong", new Class<?>[0]),
+            // Do not exist until Java 17.
+            new MethodSignature("java.util.random.RandomGenerator", "nextBoolean", new String[0]),
+            new MethodSignature("java.util.random.RandomGenerator", "nextBytes", new String[] {"byte[]"}),
```

Comment:
Similarly here, if you switch to varargs in https://github.com/jenkinsci/script-security-plugin/blob/321de82855089af63b19188356f7f713c8b92842/src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/EnumeratingWhitelist.java#L317

Reply:
Done in [f42b0b3](https://github.com/jenkinsci/script-security-plugin/pull/394/commits/f42b0b31f4df4459317e827be2e8e625e82b20f0).